app/templates/index.html

```diff
@@ -164,19 +164,15 @@ <h3 class="govuk-heading-m">{{framework.name}}</h3>
                 </li>
               {% endif %}
 
-              {% if current_user.has_role('admin-ccs-sourcing') %}
+              {% if framework['slug'] in config.PERFORMANCE_PLATFORM_ID_MAPPING and current_user.has_any_role('admin-ccs-sourcing', 'admin-framework-manager') %}
                 <li>
-                  <a class="govuk-link" href="{{ url_for('public.view_statistics', framework_slug=framework['slug']) }}">
+                  <a class="govuk-link" href="{{ config.PERFORMANCE_PLATFORM_BASE_URL }}{{ config.PERFORMANCE_PLATFORM_ID_MAPPING[framework['slug']] }}">
```

Comment:
I think Jinja will let us simplify to `config.PERFORMANCE_PLATFORM_ID_MAPPING[framework.slug]`

Reply:
I'll simplify _you_

Reply:
Fixed.

Reply:
Did it get pushed?

Reply:
Hmm. No.